generator/sbpg/targets/c.py

```diff
@@ -45,6 +45,32 @@
     "char": 1,
 }
 
+TO_GNSS = {
```

Comment:
I suggest basing this off the identifiers instead of the type names, this looks like it will let you use the existing functions `get_v4_typename` and `get_union_member_name`. So you'd have something like..

```
TO_GNSS = {
"MSG_GPS_TIME": "MSG_GPS_TIME_GNSS"
..etc..
}
```

Then below you should be able to do something like

```
        if self.name in TO_GNSS:
            self.gnss_type_name = get_v4_typename(TO_GNSS[self.name])
            self.return_union_member_name = get_union_member_name(TO_GNSS[self.name])
```

Reply:
done